ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -49,22 +81,112 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
+          /* XXX - while we define our default focus state */
+          button:focus {
+            outline: none;
+          }
+          .heading {
+            width: calc(100% - 20px);
+            overflow-x: hidden;
+            text-align: start;
+          }
+          iron-icon {
+            --iron-icon-height: 20px;
+            --iron-icon-width: 20px;
+          }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 4px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          label {
+            cursor: pointer;
+            @apply --nuxeo-label;
+          }
+          .show-more-button a {
+            display: block;
+            margin-top: 4px;
           }
         </style>
 
-        <dom-repeat items="{{buckets}}">
+        <dom-if if="[[!collapsible]]">
           <template>
-            <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
-              [[item.label]] ([[item.docCount]])
-            </paper-checkbox>
+            <dom-if if="[[_isEmpty]]">
+              <template>
+                <label>[[label]]</label>
+                <span>[[i18n('checkboxAggregation.noResults')]]</span>
+              </template>
+            </dom-if>
+
+            <dom-if if="[[!_isEmpty]]">
+              <template>
+                <label>[[label]]</label>
+                <dom-repeat items="{{buckets}}">
+                  <template>
+                    <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
+                      [[item.label]] ([[item.docCount]])
+                    </paper-checkbox>
+                  </template>
+                </dom-repeat>
+              </template>
+            </dom-if>
           </template>
-        </dom-repeat>
-        <dom-if if="[[_isEmpty]]">
+        </dom-if>
+
+        <dom-if if="[[collapsible]]">
           <template>
-            <label>[[i18n('checkboxAggregation.noResults')]]</label>
+            <dom-if if="[[_isEmpty]]">
+              <template>
+                <button aria-expanded="[[opened]]" on-tap="_toggle">
+                  <label class="heading">[[label]]</label>
+                  <iron-icon icon="[[_toggleIcon(opened)]]"></iron-icon>
+                </button>
+                <iron-collapse opened="{{opened}}">
+                  <span>[[i18n('checkboxAggregation.noResults')]]</span>
+                </iron-collapse>
+              </template>
+            </dom-if>
+
+            <dom-if if="[[!_isEmpty]]">
+              <template>
+                <button aria-expanded="[[opened]]" on-tap="_toggle">
+                  <label class="heading">[[label]]</label>
+                  <iron-icon icon="[[_toggleIcon(opened)]]"></iron-icon>
+                </button>
+                <iron-collapse opened="{{opened}}">
+                  <dom-repeat items="{{_visibleBuckets}}">
```

Comment:
Same goes here: we are duplicating
```
	        <button aria-expanded="[[opened]]" on-tap="_toggle">
                  <label class="heading">[[label]]</label>
                  <iron-icon icon="[[_toggleIcon(opened)]]"></iron-icon>
                </button>
                <iron-collapse opened="{{opened}}">
```
we we actually just want to change the contents of iron-collpase when `_isEmpty` changes.